oidc-controller/api/templates/verified_credentials.html

```diff
@@ -181,24 +183,26 @@
           <!-- If the BC_ID is clicked on 10 times in a row, display the link textbox -->
           <div v-html="svgContent" v-on:click="bcIdClicks++"></div>
         </nav>
-        <h1 class="my-2">
-          [[ headerText ]]
-        </h1>
+        <h1 class="my-2">[[ headerText ]]</h1>
         <div class="d-flex flex-row justify-content-center">
-          <status-description key="state.current" :current-state="state.current"> </status-description>
+          <status-description
+            key="state.current"
+            :current-state="state.current"
+          >
+          </status-description>
         </div>
         <div class="mx-3">
-          <error-message v-if="failed"></error-message >
+          <error-message v-if="failed"></error-message>
           <!-- BC Wallet deep link if the user agent is a mobile device -->
           <div v-if="mobileDevice && !failed">
             <a
               id="deep-link-button"
               href="{{wallet_deep_link}}"
-              type="submit"
               class="btn btn-lg btn-block btn-primary w-100 my-2 rounded-1"
               title="Open BC Wallet"
               target="_blank"
               :class="{disabled: state.disableDeeplink}"
+              v-on:click="deepLinkOpen"
```

Comment:
Add a click handler that sets up the state.
Leave the href and target blank rather than doing a window nav in the `deepLinkOpen` function. Doing it this way prevents an "allow this to open" confirmation that happens if it's done in JS.